webapp/src/components/common/datePicker/style.scss

```diff
@@ -0,0 +1,60 @@
+.rdp_wrapper {
+  position: absolute;
+  top: 44px;
+  left: 0;
+  width: auto;
+  z-index: 999;
+
+  .rdp {
+    padding: 12px;
+    border: 1px solid rgba(63, 67, 80, 0.08);
+    margin: 0;
+    background: white;
+    border-radius: 4px;
+    box-shadow: 0 8px 24px 0 rgba(0, 0, 0, 0.12);
+  }
+}
+
+.DatePicker {
+  position: relative;
+}
+
+.rdp-day {
```

Comment:
Is there any risk of these types conflicting with types specified by the web app? I'd assume that they would be the same, but I know that this is in the System Console where those types might not be applied

Reply:
Actually, if these are in the System Console, how does this look on a dark theme? Do these variables even work there?

Reply:
So these react day picker (rpd) styles are actually copy-pasted from MM webapp and only the needed parts kept, so for now, they are duplicate but they don't conflict. 

System console doesn't have a dark theme so it always uses the same colors.